packs/rackspace/actions/get_vm_ips.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: get_vm_ips
+runner_type: run-python
+description: List all the available vms by IP. Optionally filter by metadata and count
```

Comment:
Looking at the code - this action returns all the available public IP addresses for all or for a subset of VMs, right?